src/pydata_sphinx_theme/theme/pydata_sphinx_theme/components/version-switcher.html

```diff
@@ -1,5 +1,5 @@
 <div class="version-switcher__container dropdown">
-    <button type="button" class="version-switcher__button btn btn-sm navbar-btn dropdown-toggle" data-toggle="dropdown">
+    <button type="button" class="version-switcher__button btn btn-sm navbar-btn dropdown-toggle" data-bs-toggle="dropdown">
```

Comment:
version switcher dropdown is not populated on the PR build.  Is this the case for all PRs?  I haven't checked that recently.

Reply:
Ugly but it works on my side:
<img width="840" alt="Capture d’écran 2022-10-20 à 22 05 02" src="https://user-images.githubusercontent.com/12596392/197047027-607970f0-5e5b-4cae-859c-3caaa649853a.png">

Reply:
works on the homepage of the PR build but not on other pages.

Reply:
I checked the console in user_guide and the file name seems badly build, `user_guide` should not be in the url 

<img width="1225" alt="Capture d’écran 2022-10-20 à 22 26 40" src="https://user-images.githubusercontent.com/12596392/197051237-6c78bd3e-b310-4456-84a2-556c6ffab2c5.png">

Reply:
I solved the display issues and it seems the url problem is the same in every PR may be related to #647 
<img width="363" alt="Capture d’écran 2022-10-21 à 10 13 51" src="https://user-images.githubusercontent.com/12596392/197147257-269eb11e-1b61-4835-ac6e-c08b08355125.png">